code/game/objects/structures/crates_lockers/closets/secure/genpop.dm

```diff
@@ -18,6 +18,11 @@
 
 	return ..()
 
+/obj/structure/closet/secure_closet/genpop/handle_lock_addition() //If lock construction is successful we don't care what access the electronics had, so we override it

```

Comment:
I really don't like the idea that the genpop lockers are inherently different to regular ones. 

I know it's a big ask, but I think it'd be for the best if you enabled their creation using the airlock electronics in some way as well.

Reply:
It was done differently because the genpop lockers handle things with IDs, so it was necessary to check the closet had lockerelectronics before being able to close it. However, I could see if I can add the ability to create genpop lockers somehow...

Reply:
The idea would be that the genpop lockers have different, special locks. You could still unscrew them and all that jazz, but they'd need a different piece of equipment to create. Not sure what. Perhaps a setting on the airlock electronics? The things aren't any good mass produced.

Reply:
This is the last thing I need to look at before this is merge ready (I think). I'll think of some way to construct genpop lockers